WordPress/Classes/ViewRelated/NewGutenberg/NewGutenbergViewController.swift

```diff
@@ -168,9 +168,14 @@ class NewGutenbergViewController: UIViewController, PostEditor, PublishingEditor
 
         if !post.blog.isSelfHosted {
             let siteType: String = post.blog.isHostedAtWPcom ? "simple" : "atomic"
-            conf.webViewGlobals = [
-                WebViewGlobal(name: "_currentSiteType", value: .string(siteType))
-            ]
+            do {
+                conf.webViewGlobals = [
+                    try WebViewGlobal(name: "_currentSiteType", value: .string(siteType))
+                ]
+            } catch {
+                DDLogError("Failed to create WebViewGlobal: \(error)")
```

Comment:
If it's a programmer error, it should be tracked using `wpAssertionFailure` so that it appears in Sentry and is easier to catch during testing.

```suggestion
                wpAssertionFailure("Failed to create WebViewGlobal", userInfo: ["error": "\(error)"])
```

Reply:
I was unaware of this. Thank you for sharing.